widgets/DiscordappFrame.php

```diff
@@ -1,9 +1,11 @@
 <?php
+
 namespace humhub\modules\discordapp\widgets;
 
+use humhub\models\Setting;;
 use humhub\components\Widget;
 
-class DiscordappFrame extends Widget
+class Discordapp extends \humhub\components\Widget
```

Comment:
The same can go for #7 it's all a part of the plan to integrate the webhook . :+1: